keyboards/sekimen/sekimen_paw3204.c

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2019 Sekigon
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "quantum.h"
+#include "paw3204.h"
+#include "config.h"
+
+#define READ(addr) (addr)
+#define WRITE(addr) (0x80 | addr)
+
+#define REG_PID1 0x00
+#define REG_PID2 0x01
+#define REG_STAT 0x02
+#define REG_X 0x03
+#define REG_Y 0x04
+
+typedef int (*spi_paw3204_t)(uint8_t *p_tx_buffer, size_t tx_length, uint8_t *p_rx_buffer, size_t rx_length, uint8_t cs_pin);
```

Comment:
Since it looks like this uses SPI, could you convert the code to use `spi_master.c`?

Reply:
I am using the sample code of the board for trackball for Serial communication between trackball (ADTB7M) and ProMicro.

I don't fully understand the sample code.

If it is mandatory to use spi_master.c for merging into QMK, try the conversion.

But I think it will take a long time.